src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -649,5 +691,34 @@ private void EnsureMethodCanBeInvoked(string methodName)
                 throw new InvalidOperationException(Resources.FormatRazorPage_MethodCannotBeCalled(methodName));
             }
         }
+
+        // internal for testing
+        internal class TagHelperContentWrapperTextWriter : TextWriter
```

Comment:
Don't see the point in testing this. It becomes an implementation detail of `EndTagHelperWritingScope`.